wagtailvideos/views/videos.py

```diff
@@ -76,7 +75,7 @@ def index(request):
 @permission_checker.require('change')
 def edit(request, video_id):
     VideoForm = get_video_form(Video)
-
+    VideoForm.use_required_attribute = False
```

Comment:
This will also have the unintended side effect of making the `title` field not required, which is not desired.